unicodetools/src/main/java/org/unicode/idna/Uts46.java

```diff
@@ -209,12 +212,6 @@ public static boolean hasBidiError(String label, Set<Errors> errors) {
         }
 
         final int endExcludingNSM = NSM.spanBack(label, SpanCondition.CONTAINED);
-        if (endExcludingNSM == 0) {
-            // degenerate case, fails Bs 3 and 6
-            errors.add(Errors.B3);
-            errors.add(Errors.B6);
-            return true;
-        }
```

Comment:
Since the method returns early if the first character has a Bidi property other than `L`, `R`, or `AL`, we never run into this case.

Reply:
Makes sense to me. In particular:
- It does not seem to make sense to have both an LTR error and an RTL error for the same label.
- This is adding B3+B6 even though all other `errors.add(...)` are gated by LTR or RTL.